spdm_dump/spdm_dump.h

```diff
@@ -141,11 +141,12 @@ extern char *m_param_out_rsq_cert_chain_file_name;
 
 extern void *m_requester_cert_chain_buffer;
```

Comment:
please adopt same for requester

Reply:
Thanks. I have adopted same for requester cert chain buffer.